src/Jackett.Common/Utils/Extensions.cs

```diff
@@ -85,29 +85,24 @@ public static class KeyValuePairsExtension
 
     public static class ParseExtension
     {
-        public static T? TryParse<T>(this string value) where T : struct
+        public static T? TryParse<T>(this string value) where T : struct, IConvertible
```

Comment:
Could you add some unit tests for: null, empty, whitespace, "10" int, "10" double, "tt10" int .... and compare the results with the previous implementation? That is a good practice called TDD.
1. Add the unit tests first
2. The tests should pass in the old code.
3. Rewrite the code and the tests must pass in the new code.
4. All unit tests will be in place for the future changes